### PR TITLE
Pa11y test

### PR DESCRIPTION
**Pull request by simonschwartz**
This pull request adds automated accessibility testing as part of the deployment process using [pa11y-ci](https://www.npmjs.com/package/pa11y-ci).

I have also added Makefile scripts that allow developers to run accessibility tests locally.

Due to the accessibility tests running on deployment, I needed to fix accessibility issues in the current product. Most of these accessibility issues were amended in the [latest uikit mkdocs theme update](https://github.com/AusDTO/mkdocs-gov-au-theme/releases/tag/0.0.4). Some visual updates were also made to `<code>` blocks as there were colour contrast issues with the current ones.

The work in this PR was influenced by the implementation of pa11y-ci and CircleCI for the [Financial Times Orgiami project](http://jamesloveridge.net/journal/2016/using-pa11y-with-jekyll-and-circleci)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -7,13 +9,21 @@ machine:
     - cf -v
 
 dependencies:
+  pre:
+  - make pa11y-install
```
Comment (tessereth) [inline circle.yml]: Does this need to be in `pre` or can it be appended/prepended to `override`? It doesn't really matter but semantically it looks like it's setting up a dependency for the actual tests rather than something that has to happen before the other dependencies are installed.

File: docs/usage/cfcli/application_delete.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 To get a list of existing apps deployed in the curent space run `cf apps`.
 
-``` bash
+``` language-html
```
Comment (tessereth) [inline docs/usage/cfcli/application_delete.md]: I assume there's a good reason for this but this block isn't html so I'm curious why the change?

File: README.md [hunk 3]
```diff
@@ -22,3 +22,17 @@ The following assumes you have python installed locally.
 * Install dependencies with `pip install -r requirements.txt`
 * `mkdocs serve`
 * browse to http://localhost:4000
+
```
Comment (angiegove) [inline README.md:25]: Can you add that you need to have the latest mkdocs-gov-au-theme, I had a older version and the tests failed.

**Review by angiegove — COMMENTED** [review]
Looks good 👍

**Review by tessereth — APPROVED** [review]
LGTM
